lib/customisation.nix

```diff
@@ -120,6 +151,64 @@ rec {
       else result);
 
 
+  /* Call the function in the file `fn` with the required
```

Comment:
This may be a bit confusing since it's (1) named `fn` and (2) can be a function instead of a file.

Reply:
would `file or fn` be better?

Reply:
How about:
```suggestion
  /* Call the function or file containing a function with the required
```

Reply:
reworded, let me know